ui/components/component-library/avatar-token/README.mdx

```diff
@@ -0,0 +1,35 @@
+import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+
+import { AvatarToken } from './avatar-token';
+
+
+# AvatarToken
+
+The `AvatarToken` is a component responsible for display of the image of a given token
+
+<Canvas>
+  <Story id="ui-components-component-library-avatar-token-avatar-token-stories-js--default-story" />
+</Canvas>
+
+## Props
+
+The `AvatarToken` accepts all props below as well as all [BaseAvatar](/ui-components-component-library-base-avatar-base-avatar-stories-js--default-story) component props.
+
+<ArgsTable of={AvatarToken} />
+
+###  tokenName
+
+Use the ` tokenName` prop to set the initial alphabet of the `AvatarToken`.
+
+###  tokenImageUrl
+
+Use the `tokenImageUrl` prop to set the image to be rendered of the `AvatarToken`.
+
+## AvatarTokenWithHalo
+
+If we want to display the component with halo effect.
```

Comment:
Should we mention that it only works if `tokenImageUrl` is provided? Could also mention that the halo is generated by blurring the token image